---
abstract: Oysters are keystone species in the coastal environment, providing ecological, economic, and cultural services. A significant ecosystem service is their ability to improve water quality by filtration and denitrification. These ecological functions are made possible by oyster-associated microbes, but may also be threatened by microbial pathogens. This positions the oyster as a model system for the study of marine host-associated microbial diversity and function. However, studies of microbial diversity and function in shellfish are lacking, particularly in response to environmental perturbations. As coastal ecosystems change due to anthropogenic impacts and climate variability, it is important to understand how the oyster microbiome is affected and how this may impact the host. The aim of this dissertation is to determine the role of oyster associated microbiomes in response to selected environmental changes (i.e. probiotics, eutrophication, and nutrient enrichment).

Chapter 1 is a literature review describing the crosstalk between microbial community structure and function in marine host-associated microbiomes, and the importance of oyster microbiomes. The ecological need for both compositional and functional microbiome data is emphasized.

In Chapter 2, a survey of wild adult oyster gut microbiomes was performed to determine the effects of estuarine acidification and other environmental conditions. Oysters were collected at 5 sites along the north to south trophic gradient in Narragansett Bay, Rhode Island and the bacterial composition and function of their gut samples were analyzed using 16S rRNA amplicon sequencing and metatranscriptomics. Despite high variability in the bacterial community in oyster samples within each site, we found that gut bacterial communities were selected from the seawater microbiomes and varied throughout the Bay. In addition, the transcriptionally most active taxa (as detected through metatranscriptome analysis) were not the most abundant (as detected by 16S rRNA amplicon sequencing), suggesting plasticity in function as a result of redundancy. These active bacteria showed significantly increased expression of genes in stress response and phosphorus metabolism pathways at the northern, most nutrient-rich and anoxic sites, as compared to the other sites. At the southern sites, characterized by higher dissolved oxygen and lower nutrient levels, the oyster microbiomes showed a significant upregulation of genes involved in nitrogen metabolism. These shifts in microbial community composition and function inform how estuarine conditions may affect host-associated microbiomes. This research also evaluated the potential relationship between the health status of each oyster, evaluated using histology and pathogen-specific qPCR, and oyster gut microbial community composition and function (Appendix A).

The influence of nutrient enrichment on farmed adult oyster microbiomes was investigated in Chapter 3. A field study was performed at two contrasting sites in Point Judith Pond, Rhode Island, where oysters were out-planted for 3 months. Half of the oysters at each site were treated with fertilizer pellets, while the other half were maintained at ambient site conditions. Gut, inner shell, and outer shell biofilm samples were collected and analyzed using both 16S rRNA amplicon and metatranscriptomic sequencing. We detected significant differences in microbial diversity between sample types, site, and treatment (nutrient enrichment). Nutrient enrichment caused significant differential expression of nitrogen metabolism genes, but this response varied according to oyster sample type and field site. Overall nitrogen fixation and ammonia assimilation were upregulated in gut tissues, while denitrification, nitrogen fixation, and ammonia assimilation were downregulated in the outer shell samples. These results inform how oyster microbiomes perform coupled nitrification-denitrification, and how this might change with increased nutrients. In addition, Appendix B highlights the significant changes between microbial functions performed in each sample type.

In Chapter 4, we characterized bacterial community dynamics in an eastern oyster hatchery during the first 12 days after spawning and how it was affected by treatment with probiotic bacterium Bacillus pumilus RI06-95. Larvae, rearing water, and tank biofilm samples were collected from 3 separate probiotic trials and analyzed using 16S rRNA amplicon sequencing to determine the presence and relative abundances of bacteria. The bacterial community structures diverged by trial, sampling timepoint, and sample type, but there was no bulk effect of the probiotic. Instead, the probiotic acted by targeting selected taxa, amplifying Oceanospirillales in the rearing water and larvae, decreasing the relative abundance of Vibrionales, and increasing Vibrionales diversity. These targeted changes likely lead to a net decrease in potentially pathogenic species.

This dissertation emphasizes the significance of oyster-associated microbiomes and their importance to aquaculture disease prevention, wild fishery sustainability, and coastal restoration efficacy. As urbanization, coastal acidification, and disease outbreaks increase, it is important to understand these oyster-associated microbial community dynamics and how they might vary with environmental change.

authors:
- <b>Rebecca J. Stevick</b>

date: "2019-12-01T00:00:00Z"
doi: "https://digitalcommons.uri.edu/oa_diss/1089"
featured: false
image:
  focal_point: ""
  preview_only: false
publication: Open Access Dissertations
publication_types:
- "3"
publishDate: "2019-12-01T00:00:00Z"
title: 'Oyster-Associated Microbial Community Dynamics'
url_pdf: 'files/StevickDissertation2019.pdf'
citation: '**Stevick, R.J.**. "OYSTER-ASSOCIATED MICROBIAL COMMUNITY DYNAMICS" (2019). Open Access Dissertations. Paper 1089. https://digitalcommons.uri.edu/oa_diss/1089'

---



